_posts/2024-04-05-Configurable-Automation-for-OpenSearch-ML-Use-Cases.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Configurable automation for OpenSearch ML use cases"
+authors:
+    - kazabdu
+    - amitgalitz
+    - dwiddis
+    - jpalis
+    - hnyng
+    - ohltyler
+    - minalsha
+date: 2024-04-05
+categories:
+  - technical-posts
+meta_keywords: Flow Framework, OpenSearch plugins, Machine Learning
+meta_description: Explore the simplicity of integrating Machine Learning capabilities within OpenSearch through an innovative and groundbreaking framework designed to simplify complex setup tasks.
+---
+
+In OpenSearch, to use ML offerings, such as semantic, hybrid, and multimodal search, you often have to grapple with complex setup and preprocessing tasks. Additionally, you must write verbose queries, which can be time-consuming and error-prone.
+
+In this blog post, we introduce the OpenSearch Flow Framework plugin, released in version 2.13, designed to streamline this cumbersome process. By using this plugin, you can simplify complex setups with just one click. We've provided automated templates, enabling you to create connectors, register models, deploy them, and register agents and tools through a single API call. This eliminates the complexity of calling multiple APIs and orchestrating setups based on the responses.
```

Comment:
Please add a link here: "released in version 2.13" to the release blog or release notes. 
https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.13.0.md